state-chain/runtime/src/lib.rs

```diff
@@ -1070,7 +1070,7 @@ impl_runtime_apis! {
 		fn cf_account_flip_balance(account_id: &AccountId) -> u128 {
 			pallet_cf_flip::Account::<Runtime>::get(account_id).total()
 		}
-		fn cf_account_info_v2(account_id: &AccountId) -> RuntimeApiAccountInfoV2 {
+		fn cf_validator_info(account_id: &AccountId) -> RuntimeApiAccountInfoV2 {
```

Comment:
If `RuntimeApiAccountInfoV2` is now only used for validators, we could rename it as `ValidatorInfo` to be consistent with other naming. 

As a next step (I'll open issue) we could factor out duplicate information and have a struct like `AccountInfo<Role>` where the role defines the additional information for each role. Something like this:

```rust
struct AccountInfo<RoleInfo> {
	balance: NumberOrHex,
	#[serde(flatten)]
	role_info: RoleInfo,
}

enum RpcAccountInfo {
	Unregistered(AccountInfo<()>),
	Lp(AccountInfo<LpAccountInfo>),
	//..
}
```

It should be possible to do this without chaning the encoding, and it will make things easier to maintain.

Reply:
will do, sounds good!